resources/public/api/definition.json

```diff
@@ -18,7 +18,18 @@
         "endpointsEnabled": true,
         "access": {
           "type": "PUBLIC"
-        }
+        },
+        "fieldDefinitions": [{
+          "name": "notificationUrl",
+          "description": "What is your notification web address for us to send push notifications to?",
+          "type": "URL",
+          "hint": "You must only give us a web address that you own. Your application will use this address to listen to notifications from HMRC."
+        }, {
```

Comment:
I think there should be validation rules for this field similar to https://github.com/hmrc/goods-movement-system-haulier-api/blob/master/app/uk/gov/hmrc/goodsmovementsystemhaulierapi/platform/views/definition.scala.txt#L30